tests/kernels/test_machete_gemm.py

```diff
@@ -0,0 +1,168 @@
+"""Tests for the marlin kernel.
+
+Run `pytest tests/kernels/marlin/test_machete_gemm.py`.
+"""
+
+import math
+from typing import Optional, Tuple
+
+import pytest
+import torch
+
+from vllm import _custom_ops as ops
+from vllm.model_executor.layers.quantization.utils.quant_utils import (
+    pack_rows, quantize_weights)
+from vllm.platforms import current_platform
+from vllm.scalar_type import ScalarType, scalar_types
+
+MNK_SHAPES = [
+    (1, 128, 128),
+    (1, 512, 1024),
+    (1, 4096, 4096),
+    (13, 8192, 4096),
+    (26, 4096, 8192),
+    (1, 4096, 4096),
+    (257, 128, 4096),
+    (257, 4224, 4096),
+    (257, 4096, 4096),
+    (64, 4096, 4096),
+]
```

Comment:
could you add a couple of nastier sizes here? (see also my question about adding TORCH_CHECKs for divisibility constraints)

Reply:
hmmm what would you consider nastier where N is still multiple of 128 and K is a still multiple of 64?

Reply:
I wasn't sure what the divisibility constraints were (see my other comment :smile: ) but this helps, thanks